logger.go

```diff
@@ -121,6 +129,22 @@ func (l *Logger) output(s severity, txt string) {
 	}
 }
 
+// Close closes all the underlying log destinations, which will flush all the
+// cached logs. Pleaes call Close once all logging is done. This is commonly
+// setup as a deferred function call in main after Init.
+// Once Close is called, all future calls to the logger will panic. It will
+// print a warning to stderr for ervery log destination that is failed to close,
```

Comment:
This wont return nil, as in a nil error, it doesn't return anything.

It will print a warning to stderr for every log destination errors on close.

Reply:
Done